src/utils/string/StringUtils.js

```diff
@@ -64,9 +64,8 @@ const detectStateInCity = (city) => {
  * @returns {String}
  */
 const displayCityName = (city, state) => {
-  if (city === null || city === undefined || city === '') throw new Error('City must be defined');
-  if (state === null || state === undefined || state === '')
-    throw new Error('State must be defined');
+  if (city === null || city === undefined || city === '') return null;
+  if (state === null || state === undefined || state === '') return null;
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The new code returns `null` when either `city` or `state` is not provided. This could potentially lead to unexpected behavior if the calling function does not handle `null` values properly. Consider returning a default value or throwing an error instead.
```diff
-  if (city === null || city === undefined || city === '') return null;
-  if (state === null || state === undefined || state === '') return null;
+  if (!city) throw new Error('City must be defined');
+  if (!state) throw new Error('State must be defined');
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->